server/port.js

```diff
@@ -1,3 +1,3 @@
 const argv = require('./argv');
 
-module.exports = parseInt(argv.port || process.env.PORT || '3000', 10);
+module.exports = parseInt(argv.port || process.env.PORT || '8000', 10);
```

Comment:
Why changing the port?

Reply:
The  port 3000  likely to be occupied....